src/libs/actions/connections/SageIntacct.ts

```diff
@@ -0,0 +1,15 @@
+import * as API from '@libs/API';
+import type ConnectPolicyToSageIntacctParams from '@libs/API/parameters/ConnectPolicyToSageIntacctParams';
+import {READ_COMMANDS} from '@libs/API/types';
+
+function connectToSageIntacct(policyID: string, companyID?: string, userID?: string, password?: string) {
+    const parameters: ConnectPolicyToSageIntacctParams = {
+        policyID,
+        intacctCompanyID: companyID,
+        intacctUserID: userID,
+        intacctPassword: password,
+    };
+    API.read(READ_COMMANDS.CONNECT_POLICY_TO_SAGE_INTACCT, parameters, {});
```

Comment:
Is it read command? Shouldn't this be write command?

Reply:
All Xero/QBO commands are READ_COMMANDS, that's why I put CONNECT_POLICY_TO_SAGE_INTACCT into READ_COMMANDS. Anyways, whats the difference between those two? Connecting action works with both cases 🤣

Reply:
I am not sure to be honest. I thought all form actions should be WRITE_COMMANDS. I think my assumption could be wrong here then. @yuwenmemon can chime in.